### PR TITLE
Add related to topic "gRPC"

### PR DESCRIPTION
**Pull request by shaedrich**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I am changing.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

Other RPC-based stuff

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/grpc/index.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ short_description: gRPC is a high performance RPC framework.
 topic: grpc
 url: https://grpc.io/
 wikipedia_url: https://en.wikipedia.org/wiki/GRPC
+related: rpc, http2, protobuf, capn-proto, json-rpc
 ---
-gRPC is a modern open source high performance RPC framework that can run in any environment. It can efficiently connect services in and across data centers with pluggable support for load balancing, tracing, health checking and authentication. It is also applicable in last mile of distributed computing to connect devices, mobile applications and browsers to backend services.
+**gRPC** (**gRPC Remote Procedure Calls**) is a modern open source high performance RPC framework that can run in any environment. It can efficiently connect services in and across data centers with pluggable support for load balancing, tracing, health checking and authentication. It is also applicable in last mile of distributed computing to connect devices, mobile applications and browsers to backend services.
```
Comment (kenyonj) [inline topics/grpc/index.md:13]: It seems odd here that the acronym definition contains the acronym again. Is that intentional?

Comment (shaedrich) [reply topics/grpc/index.md:13]: It actually is. It seems to be a recursive acronym—that's at least what the website says. Should this be emphasized textually?

Comment (kenyonj) [reply topics/grpc/index.md:13]: OK then 😆! I think it's fine the way you have it.

**Review by kenyonj — APPROVED** [review]
LGTM!

File: topics/grpc/index.md (hunk 1 above)
Comment (shaedrich) [reply topics/grpc/index.md:13]: Yeah, I wondered about that, too. But if they say so, I thought "be my guest" 😆

**Pull request merged**